examples/ecs/src/components.cairo

```diff
@@ -1,11 +1,53 @@
 use array::ArrayTrait;
+use core::debug::PrintTrait;
 use starknet::ContractAddress;
 
+#[derive(Serde, Copy, Drop)]
+enum Direction {
+    None: (),
+    Left: (),
+    Right: (),
+    Up: (),
+    Down: (),
+}
+
+impl DirectionPrintImpl of PrintTrait<Direction> {
```

Comment:
For some reason, running `sozo test` something is forcing debug print of all component members so needed to implement PrintTrait, otherwise can be removed

Reply:
Hmm I'll take a look at this